server/data/ApiData.ts

```diff
@@ -0,0 +1,11 @@
+export type ApiPageResponse<T> = {
```

Comment:
Moved out of specific Response files as these are generic for Spring APIs

Reply:
Cool!